block parse failure for mainnet block 12c1f09d72c232659022fc67ab2bbe1a6db498817d0dd1d751dd6acf5ae1aa38

We get a panic from deep-ish in the CBOR library.

```
panic: runtime error: hash of unhashable type []interface {}
```

https://cexplorer.io/block/12c1f09d72c232659022fc67ab2bbe1a6db498817d0dd1d751dd6acf5ae1aa38

```
$ ./block-fetch -address relays-new.cardano-mainnet.iohk.io:3001 -network mainnet -ntn -slot 4563840 -hash 12c1f09d72c232659022fc67ab2bbe1a6db498817d0dd1d751dd6acf5ae1aa38
panic: runtime error: hash of unhashable type []interface {}

goroutine 28 [running]:
reflect.mapassign(0x670120, 0xc000198c50?, 0x0?, 0x6b9312?)
        /usr/local/go/src/runtime/map.go:1351 +0x25
reflect.Value.SetMapIndex({0x670120?, 0xc00019a2b0?, 0xc000198c50?}, {0x67c860, 0xc000198be0, 0x199}, {0x67c860, 0xc000198c50, 0x199})
        /usr/local/go/src/reflect/value.go:2255 +0x391
github.com/fxamacker/cbor/v2.(*decoder).parseMapToMap(0xc000311f30, {0x670120?, 0xc00019a2b0?, 0x0?}, 0xc000196d20)
        /home/agaffney/go/pkg/mod/github.com/fxamacker/cbor/v2@v2.4.0/decode.go:1295 +0x9a9
github.com/fxamacker/cbor/v2.(*decoder).parseToValue(0xc000311f30, {0x670120?, 0xc00019a2b0?, 0xc00031a000?}, 0x0?)
        /home/agaffney/go/pkg/mod/github.com/fxamacker/cbor/v2@v2.4.0/decode.go:782 +0xef8
...
```

--- a/ledger/shelley.go
+++ b/ledger/shelley.go
@@ -111,7 +111,7 @@ type ShelleyTransactionBody struct {
 	Fee     uint64                     `cbor:"2,keyasint,omitempty"`
 	Ttl     uint64                     `cbor:"3,keyasint,omitempty"`
 	// TODO: figure out how to parse properly
-	Certificates []cbor.Value `cbor:"4,keyasint,omitempty"`
+	Certificates cbor.RawMessage `cbor:"4,keyasint,omitempty"`
 	// TODO: figure out how to parse this correctly
 	// We keep the raw CBOR because it can contain a map with []byte keys, which
 	// Go does not allow

